src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentAjaxPageData.java

```diff
@@ -44,10 +47,9 @@ public FeedbackResponseCommentRow getComment() {
                 FeedbackParticipantType.STUDENTS,
                 FeedbackParticipantType.INSTRUCTORS
         };
-
         Map<FeedbackParticipantType, Boolean> responseVisibilities = new HashMap<>();
         for (FeedbackParticipantType type : relevantTypes) {
-            responseVisibilities.put(type, true);
+            responseVisibilities.put(type, question.isResponseVisibleTo(type));
```

Comment:
The above line fixes the problem stated in the [#comment](https://github.com/TEAMMATES/teammates/pull/7418#issuecomment-308003304) in PR 7418.